root_orchestrator/system-manager-python/services/service_management.py

```diff
@@ -44,48 +67,58 @@ def create_services_of_app(username, sla, force=False):
     return {"job_id": str(last_service_id)}, 200
 
 
+def delete_job(job_id):
+    logging.info("delete job...")
+    job_operations.delete_job(job_id)
+
+
 def delete_service(username, serviceid):
-    apps = mongo_get_applications_of_user(username)
+    apps = app_operations.get_user_apps(username)
     for application in apps:
         if serviceid in application["microservices"]:
-            # undeploy instances
             request_scale_down_instance(serviceid, username)
-            # remove service from app's services list
             remove_service_from_app(application["applicationID"], serviceid, username)
-            # remove service from DB
-            mongo_delete_job(serviceid)
-            # inform network component
+            delete_job(serviceid)
             net_inform_service_undeploy(serviceid)
             return True
     return False
 
 
 def update_service(username, sla, serviceid):
     # TODO Check fields and redeploy service
```

Comment:
Exactly! Can we temporarily return error with this method? I noticed that in the current version in develop and main it creates more issues than solutions. Until we have this check this looks to me as we're just creating issues.